app/service/statistics.py

```diff
@@ -84,7 +84,8 @@ def create_zeroed_stats_dicts():
 
 
 def _update_statuses_from_row(update_dict, row):
-    update_dict['requested'] += row.count
+    if row.status is not 'cancelled':
```

Comment:
This line needs to be `!= 'cancelled` instead of `is not 'cancelled'` in order to show the right numbers in Admin

Reply:
Hahah, I had a one-line job and I blundered, that's cute xD. Now I read up and I even know why:
"The keyword `is` relates to two objects being the exact same object, whereas `==` and `!=` deal with two objects being equivalent but not necessarily the same object"

Thanks for thorough testing 💖.

Reply:
Confusingly the test was making it look like the version with `is not 'cancelled'` _was_ working since Python is caching the 'cancelled' string object and using the exact same object in the test and the function 🤯

Reply:
Wow! 😮